### PR TITLE
AMIgo auto updates

### PR DESCRIPTION
**Pull request by sihil**
Move to an AMIgo image. Includes changes to YAML CFN and debian packaging.

### CODE REVIEW

**Comment by sihil** [comment]
Deployed on CODE

File: project/plugins.sbt [hunk 1]
```diff
@@ -5,3 +5,6 @@ addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.4.6")
 
 addSbtPlugin("com.gu" % "sbt-riffraff-artifact" % "0.9.7")
 
+addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.0-RC10")
```
Comment (akash1810) [inline project/plugins.sbt:8]: nice!

**Review by akash1810 — APPROVED** [review]
lgtm

**Comment by sihil** [comment]
[Path Manager: Update AMI / Riff-Raff](https://trello.com/c/btRInZAq/185-path-manager-update-ami-riff-raff)

**Pull request merged**